1_prep/src/munge_obs.R

```diff
@@ -2,32 +2,47 @@
 #'
 #' @param data chr or object, if chr use full file path for the complete dataset
 #' @param site_id chr, site id for the lake of interest
+#' @param remove_dups logical, should duplicate instances of depth and date be removed?
 #' @param path_out chr, subfolder for writing the lake-specific dataset
 #'
-subset_model_obs_data <- function(data, site_id, path_out = '1_prep/out/field_data_all') {
+subset_model_obs_data <- function(data, site_id, remove_dups = FALSE, path_out = '1_prep/out/field_data_all') {
```

Comment:
Yes, it does. I added this option in because duplicate values are only a problem in the repaired data, but I use this function to create `p1_obs_rds` (which uses the de-duplicated `merged_temp_data_daily.feather`)  and `p1_obs_buffer_from_dam_rds` (which uses raw cooperator data).